### DISCUSSION TITLE
Quick create toggleable

### DISCUSSION THREAD

**Discussion by Hatzefatz** [QuickShop-Community/QuickShop-Hikari#1390] [Ideas]
I think it would be nice to have that feature, but not always on. So the idea is, to create a command, which makes the quick create toggleable. And to make it perfect, create a configuration for it, to set the default to enabled or disabled.

**Comment by creatorfromhell** [comment]
> I think it would be nice to have that feature, but not always on. So the idea is, to create a command, which makes the quick create toggleable. And to make it perfect, create a configuration for it, to set the default to enabled or disabled.

Sorry for the late response, what's the benefit of this?

**Reply by Hatzefatz** [reply]
Hi, No problem at all!

To be honest, I''m not sure anymore, what i was requesting that for... I'm not in the MC bubble anymore at the moment. If i remember, i will get back to you!

Best,
Hatzefatz
